core/trino-spi/src/main/java/io/trino/spi/procedure/Procedure.java

```diff
@@ -99,14 +99,12 @@ public boolean requiresNamedArguments()
     @Override
     public String toString()
     {
-        return new StringBuilder()
-                .append(schema).append('.').append(name)
-                .append('(')
-                .append(arguments.stream()
+        return schema + '.' + name +
+                '(' +
+                arguments.stream()
```

Comment:
why not stringjoiner?

Reply:
It was just a mechanical transformation. If I was taking it further I would rather use `Collectors#joining(delim, prefix, suffix)` instead.